packages/dashboard/src/components/dashboard/index.tsx

```diff
@@ -48,7 +48,7 @@ const Dashboard: React.FC<DashboardProperties> = ({
                 enableKeyboardEvents: true,
               }}
             >
-              <InternalDashboard onSave={onSave} editable={true} propertiesSections={propertiesSections} />
+              <InternalDashboard onSave={onSave} editable={true} propertiesPanel={<PropertiesPanel />} />
```

Comment:
we are passing this as a prop because we want to render this only in edit mode right ? 
why not just check the redux state variable which tells that and let <PropertiesPanel> be just a part within InternalDashboard ? 
something like 
`edit ? <ShowPanel> : <>`